sheets/src/main/java/com/novoda/github/reports/sheets/network/SheetsServiceClient.java

```diff
@@ -0,0 +1,40 @@
+package com.novoda.github.reports.sheets.network;
+
+import com.novoda.github.reports.sheets.sheet.Entry;
+import com.novoda.github.reports.sheets.sheet.Sheet;
+
+import retrofit2.Response;
+import rx.Observable;
+import rx.functions.Func1;
+
+public class SheetsServiceClient {
+
+    // TODO extract to .json and read as prop
+    private static final String DOCUMENT_ID = "1rMeGnlugO312to0loBwN3x0QTvAxoHwv4Pe_SYXR1YE";
+
+    private final SheetsApiService apiService;
+
+    public static SheetsServiceClient newInstance() {
+        SheetsApiService apiService = SheetsServiceContainer.getSheetsService();
+        return new SheetsServiceClient(apiService);
+    }
+
+    SheetsServiceClient(SheetsApiService apiService) {
+        this.apiService = apiService;
+    }
+
+    public Observable<Sheet> getDocument() {
+        return apiService.getDocument(DOCUMENT_ID)
+                .map(Response::body);
+    }
+
+    public Observable<Entry> getEntries() {
+         return apiService.getDocument(DOCUMENT_ID)
+                 .flatMap(toEntries());
```

Comment:
yeah it is only small :shrug:

would still watch the talk its good! http://uk.droidcon.com/skillscasts/8416-making-retrofit-2-work-for-you shows you a good Separation of Concerns